src/pages/student_council.tsx

```diff
@@ -0,0 +1,173 @@
+                                alt={card.title}
+                            />
+                            <div className="opacity-0 group-hover:opacity-100 transition-opacity duration-200 fixed text-center left-0 right-0 bottom-0 text-white p-4">
+                                <h3 className="md:text-xl text-xs font-bold">{card.title}</h3>
+                                <p className="md:text-base text-xs font-bold">{card.description}</p>
+                            </div>
+                        </div>
+                    </div>
+                ))}
+            </div>
+
+            <div className="flex items-center justify-center">
+                <UnstyledButton onClick={() =>
+                    scrollIntoView({
+                        alignment: 'center',
+                    })
+                }>
+                    <ThemeIcon radius="xl" className="bouncy">
+                        <FaArrowDown />
+                    </ThemeIcon>
+                </UnstyledButton>
+            </div>
+
+            <div className="container mx-auto pt-80" ref={targetRef}>
+                <div className="flex lg:flex-row flex-col lg:space-y-0 space-y-4 justify-between w-full">
+                    <Paper className="flex md:w-[47%] md:h-[27rem] h-72 w-full rounded-lg">
+                        <div className="relative">
+                            <h1 className="md:pl-6 md:text-4xl pl-3 text-xl">About</h1>
+                            <p className="md:pl-6 pl-3 md:w-[350px] w-[250px] text-base">
+                                The Student Council is the official student body of Stamford International University. The
+                                council is responsible for representing the student body, organizing events, and
+                                communicating with the university&apos;s administration.
+                            </p>
+                            <div className="absolute top-0 md:ml-[380px] ml-[280px] md:mt-7 mt-4">
+                                <Image className="md:w-80 md:h-96 w-52 h-64 rounded-lg object-cover" src="/assets/images/student_council/group.jpg" alt="" />
+                            </div>
+                        </div>
+                    </Paper>
+                    <Paper className="flex md:w-[47%] md:h-[27rem] h-72 w-full rounded-lg">
+                        <div className="relative">
+                            <h1 className="md:pl-6 md:text-4xl pl-3 text-xl">Upcoming Event</h1>
+                            <p className="md:pl-6 pl-3 md:w-[350px] w-[250px] text-base">
+                                The "Welcome Freshmen" event, orchestrated by the Student Council at Stamford International University,
+                                symbolizes the warm embrace awaiting new students. This event serves as an introduction to the vibrant community,
+                                showcasing the Council's dedication to fostering a welcoming environment and facilitating a
```

Comment:
`'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`

Reply:
You escape these because:
- Certain characters need to be escaped to be displayed correctly
- Consistency
- Avoiding Accidental Interpretation / Syntax Errors